examples/gno.land/p/demo/pm/pm.gno

```diff
@@ -0,0 +1,54 @@
+// On-chain project management utilities.
+// This library enables efficient project management for evaluation DAOs, governance modules, community pool management, launchpads, or general transparency.
+package pm
+
+// ITask is a completion unit that is associated with IOwner and can include sub-tasks.
+// It is typically utilized for issues, epics, milestones, cards, and stories.
+type ITask interface {
+	// attributes
+
+	ID() string
+	Title() string
+	// Kind could be Issue, PR, Milestone, Epic, Story, Card, ...
+	Kind() stringOrEnum
+	State() Enum
```

Comment:
`Status()` instead of` State()`, if you mean something like pending, completed etc